Installation/changes.html

```diff
@@ -129,6 +129,26 @@
 </table>
 
 <HR>
+<h2 id="release4.12">Release 4.12 </h2>
+ <div>
+   <p>Release date: Mars 2018 </p>
+ 
+  <h3>3D Surface Mesh Generation</h3>
+   <ul>
+     <li>
+       Add a function to export c2t3 facets into a facegraph.
```

Comment:
please name the function so that the user do not have to look for it + I guess you meant `MutableFaceGraph` + please use the correct name for `c2t3`.